server/src/commandLine.mjs

```diff
@@ -122,5 +124,6 @@ export {
   httpPort, socketPort,
   conduitSocketPort, conduitKeySocketPort,
   developerToolPort, developerToolName,
-  enabledSdkNames, enabledTriggerPaths, proxy
+  enabledSdkNames, enabledTriggerPaths, proxy,
+  conduit
```

Comment:
readme for conduit command line passing is missing. What command we need to run to start conduit?